go.mod

```diff
@@ -19,7 +19,7 @@ require (
 	github.com/blevesearch/bleve/v2 v2.2.2
 	github.com/coreos/go-oidc/v3 v3.1.0
 	github.com/cs3org/go-cs3apis v0.0.0-20211018122138-391b29bd7803
-	github.com/cs3org/reva v1.15.1-0.20211027114107-4879bf6be97a
+	github.com/cs3org/reva v1.15.1-0.20211108144647-80f5ec9b8978
```

Comment:
Changelog says it is bumped to `v1.16` but this still says `v1.15`. What am I missing?

Reply:
REVA  1.16 is not yet released and the changelog item is just a placeholder